cli/command/run/cmd.go

```diff
@@ -531,6 +536,68 @@ func runTestcafeInCloud(p testcafe.Project, regio region.Region, tc testcomposer
 	return r.RunProject()
 }
 
+func runXcuit(cmd *cobra.Command, tc testcomposer.Client, rs resto.Client, rc rdc.Client, as *appstore.AppStore) (int, error) {
+	p, err := xcuit.FromFile(cfgFilePath)
+	if err != nil {
+		return 1, err
+	}
+	p.Sauce.Metadata.ExpandEnv()
+	applyDefaultValues(&p.Sauce)
+	overrideCliParameters(cmd, &p.Sauce)
+
+	// TODO - add dry-run mode
+	regio := region.FromString(p.Sauce.Region)
+	if regio == region.None {
+		log.Error().Str("region", regionFlag).Msg("Unable to determine sauce region.")
+		return 1, errors.New("no sauce region set")
+	}
+
+	err = xcuit.Validate(p)
+	if err != nil {
+		return 1, err
+	}
+
+	if cmd.Flags().Lookup("suite").Changed {
+		if err := filterXcuitSuite(&p); err != nil {
+			return 1, err
+		}
+	}
+
+	tc.URL = regio.APIBaseURL()
+	rs.URL = regio.APIBaseURL()
+	as.URL = regio.APIBaseURL()
+	rc.URL = regio.APIBaseURL()
+
+	rs.ArtifactConfig = p.Artifacts.Download
+	rc.ArtifactConfig = p.Artifacts.Download
+
+	return runXcuitInCloud(p, regio, tc, rs, rc, as)
+}
+
+func runXcuitInCloud(p xcuit.Project, regio region.Region, tc testcomposer.Client, rs resto.Client, rc rdc.Client, as *appstore.AppStore) (int, error) {
+	log.Info().Msg("Running Xcuitest in Sauce Labs")
```

Comment:
```suggestion
	log.Info().Msg("Running XCUITest in Sauce Labs")
```
We should keep the original way to write the name.